lib/lotus/commands/db.rb

```diff
@@ -0,0 +1,59 @@
+require 'lotus/router'
+require 'lotus/model'
+
+module Lotus
+  module Commands
+    class DB
+      attr_reader :environment, :options, :name
+      DEFAULTS = { step:1 }
+
+      def initialize(name=nil,environment)
+        @name = name
+        @environment = environment
+        @options = merge_options(@environment)
```

Comment:
what do you think about instantiate a migrator object in the constructor? `@migrator = Lotus::Model::Migrator.new(adapter_config)`